pkg/mock/fileUtils.go

```diff
@@ -84,27 +118,26 @@ func (f *FilesMock) HasWrittenFile(path string) bool {
 // FileExists returns true if file content has been associated with the given path, false otherwise.
 // Only relative paths are supported.
 func (f *FilesMock) FileExists(path string) (bool, error) {
-	if f.files == nil {
-		return false, nil
-	}
```

Comment:
Why can you delete this? The map may not be initialized. You don't have to initialize it for FileExists(), but you still need to check if it is.

Reply:
Restored. I removed it and all the tests succeeded as well. Basically it looks bogus to me to act upon an uninitialized instance. In case we encounter that situation wouldn't it be better to answer with an error accordingly  outlining that we work upon an uninitialized instance. That would be encountered very soon during developing a test and can be fixed. When I see a method like `init` I assume implicitly that this method needs to be called prior to using the instance (like a constructor).